vscode-extension/package.json

```diff
@@ -110,44 +110,41 @@
     "walkthroughs": [
       {
         "id": "vscode-aiconfig.welcomeWalkthrough",
-        "title": "AIConfig Editor Getting Started",
-        "description": "Your first steps to using AIConfig Editor - the generative AI studio for VS Code!",
+        "title": "Get Started with Generative AI",
```

Comment:
nit: Would prefer "Get Started with AIConfig" instead since "Generative AI" feels super general, not a blocker

Reply:
The reason I changed it was because it already says "AIConfig Editor" in the command. Will update back